python/amici/petab_objective.py

```diff
@@ -35,7 +35,6 @@ def simulate_petab(
         problem_parameters: Optional[Dict[str, float]] = None,
         simulation_conditions: Union[pd.DataFrame, Dict] = None,
         parameter_mapping: List[petab.ParMappingDictTuple] = None,
```

Comment:
Now expects a different type, right?

Reply:
right. is the design ok for you, i.e. to only have one thing passed now?

Reply:
Makes sense now, I'd say.